drivers/adc/adc_smartbond_sdadc.c

```diff
@@ -124,6 +171,7 @@ static void adc_context_start_sampling(struct adc_context *ctx)
 		/* TODO: Add calibration code */
 	} else {
 		val = SDADC->SDADC_CTRL_REG & ~PER_CHANNEL_ADC_CONFIG_MASK;
+		val |= SDADC_SDADC_CTRL_REG_SDADC_EN_Msk;
```

Comment:
We should wait for SDADC_LDO_OK before start sampling. Should we move it back to `sdadc_smartbond_resume()`?

Reply:
Good point, moved it back